### PR TITLE
ISPN-10508 Server Docs: Getting Started

### PR DESCRIPTION
**Pull request by oraNod**
https://issues.jboss.org/browse/ISPN-10508

### CODE REVIEW

File: documentation/src/main/asciidoc/topics/proc_creating_cache_rest.adoc [hunk 1]
```diff
@@ -0,0 +1,36 @@
+= Creating Caches
+{brandname} servers do not provide default caches. To store data in your cluster, you must first define a cache.
+
+.Prerequisites
+A REST client. The following procedures use `curl` but you can use any client.
```
Comment (tristantarrant) [inline documentation/src/main/asciidoc/topics/proc_creating_cache_rest.adoc]: As soon as we merge the CLI, this paragraph can be replaced with instructions using that

**Comment by tristantarrant** [comment]
Can you rebase this @oraNod

**Comment by tristantarrant** [comment]
Needs rebase

**Comment by oraNod** [comment]
@tristantarrant rebased. thanks!

**Comment by tristantarrant** [comment]
Merged

**Pull request merged**